### DISCUSSION TITLE
Praise for the Langfuse library!  🚀

### DISCUSSION THREAD

**Discussion by ttamg** [langfuse/langfuse#2817] [Support] [answered]
I wanted to say that the client and the libraries (we are using Python) are excellent already out of the box.   💯 

The langfuse team have done a fantastic job, and it is great that there is an open source option too.  I can't praise what you all have done enough.  Have looked at a number of other systems, many vended and expensive and this is way ahead in terms of architecture, features and just usability.   Love it!

We will very happily contribute too if useful.

Thanks! 🙏

**Comment by marcklingen** [comment, marked as answer]
thank you @ttamg!
